<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QMainWindow" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>remote</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="1" column="0">
         <layout class="QGridLayout" name="tabGL">
          <item row="4" column="0" rowspan="2">
           <widget class="QGroupBox" name="remoteConGB">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QLabel" name="lport">
               <property name="text">
                <string>port</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lip">
               <property name="text">
                <string>IP</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lkey">
               <property name="text">
                <string>key</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="keyEdit">
               <property name="text">
                <string/>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="3">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="linkButton">
                 <property name="text">
                  <string>連線</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="stopButton">
                 <property name="text">
                  <string>停止</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="portEdit">
               <property name="text">
                <string>7777</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="ipEdit">
               <property name="text">
                <string>127.0.0.1</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <layout class="QHBoxLayout" name="commandHBL">
            <item>
             <widget class="QLabel" name="lcmd">
              <property name="text">
               <string>  命令:  </string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cmdEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="cmdlinkButton">
              <property name="text">
               <string>傳送命令</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QTextEdit" name="remoteLogTextEdit">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="tabStopWidth">
             <number>160</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1" rowspan="2">
           <widget class="QGroupBox" name="MCLogsConfigGB">
            <property name="title">
             <string>MCLogsConfig(msec)</string>
            </property>
            <layout class="QGridLayout" name="logCfgGL">
             <item row="5" column="0" colspan="3">
              <layout class="QHBoxLayout" name="UpdateCBHBL">
               <item>
                <widget class="QCheckBox" name="MCStatusUpdateCB">
                 <property name="text">
                  <string>Status自動更新</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="MCLogsUpdateCB">
                 <property name="text">
                  <string>Logs自動更新</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lMCLogs">
               <property name="text">
                <string>Logs</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="MCStatusMSecLE">
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>1000</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="MCLogsMSecLE">
               <property name="placeholderText">
                <string>5000</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lMCStatus">
               <property name="text">
                <string>Status</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="2" rowspan="2">
           <widget class="QGroupBox" name="remoteCtlGB">
            <property name="font">
             <font>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="4" column="0" colspan="3">
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="0">
                <widget class="QPushButton" name="MCLogsRefreshButton">
                 <property name="text">
                  <string>取得server logs</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="remoteStartButton">
                 <property name="text">
                  <string>遠端開啟</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="remoteStopButton">
                 <property name="text">
                  <string>遠端停止</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="statusRefreshButton">
                 <property name="text">
                  <string>伺服器狀態更新</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="getFileButton">
                 <property name="text">
                  <string>取得server log 檔</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>mc logs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="tab_2GL">
          <item row="0" column="0">
           <widget class="QTextEdit" name="mcServerLogTextEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
